yarn.lock

```diff
@@ -714,14 +714,15 @@
   resolved "https://registry.npmjs.org/@sindresorhus/is/-/is-0.7.0.tgz#9a06f4f137ee84d7df0460c1fdb1135ffa6c50fd"
   integrity sha512-ONhaKPIufzzrlNbqtWFFd+jlnemX6lJAgq9ZeiZtS7I1PIf/la7CW4m83rTXRnVnsMbW2k56pGYu7AUFJD9Pow==
 
-"@sourcegraph/basic-code-intel@^6.0.7":
-  version "6.0.7"
-  resolved "https://registry.npmjs.org/@sourcegraph/basic-code-intel/-/basic-code-intel-6.0.7.tgz#6ba8623c1c5e71f780a059ccea2120ffdc1e3600"
-  integrity sha512-+amGIq54N3aVOj7AxNtW6QtDfUVvtT253Ha+i9sofyo217L/7Ucwo/WoDGuded9Ng9FUxUM6UOS3MREW+lkUGQ==
+"@sourcegraph/basic-code-intel@^6.0.12":
+  version "6.0.12"
+  resolved "https://registry.npmjs.org/@sourcegraph/basic-code-intel/-/basic-code-intel-6.0.12.tgz#5d6c1a301c73a19bf28f6d25b5f4846cc3db844f"
+  integrity sha512-G8IUx29UV1asiU3SZ3AMYKlZSZxX63nC7IR4ux+Jd+TBg2OW9WrRKOEaYKp2MWlwyWBc7Dpix3hv6GqMSMJNyA==
   dependencies:
     lodash "^4.17.11"
     rxjs "^6.3.3"
     sourcegraph "^23.0.0"
+    sprintf-js "^1.1.2"
```

Comment:
I'm curious what `sprintf-js` is used for? Never seen anyone use sprintf syntax in JS since we have template strings and arrow functions to define templates

Reply:
I forgot to remove sprintf-js after removing definition patterns, which used the `%s` syntax. Removing from basic-code-intel in https://github.com/sourcegraph/sourcegraph-basic-code-intel/pull/80